modules/lily-compiler/src/test/java/io/github/tomboyo/lily/compiler/CompilerSupport.java

```diff
@@ -145,7 +145,7 @@ private static String getLinesAround(String source, long fromLong, long toLong)
     // Move pointers to the beginning and end of their respective lines if they aren't already there
     // index may be -1 if absent, so we Max to 0.
     from = Math.max(0, source.lastIndexOf('\n', from));
-    to = Math.max(0, source.indexOf('\n', to));
+    to = Math.max(source.length(), source.indexOf('\n', to));
```

Comment:
This is wrong, this always goes to source length. Use `to` if there is no newline, else the index of the following newline.